F/FMM3D/build_tarballs.jl

```diff
@@ -34,16 +34,18 @@ touch make.inc
 # force gcc
 if [[ ${target} = *apple* || ${target} = *freebsd* ]]; then
   export CC="gcc"
+  export CXX="g++"
+  export FC="gfortran"
 fi
```

Comment:
Why are we forcing GCC at all?  Also, `FC` is _already_ `gfortran`.

Reply:
I'll see if this is still necessary. Previously, some of the other compilers would fail for reasons that weren't clear.

Reply:
That was probably before llvm openmp was available: https://github.com/JuliaPackaging/Yggdrasil/blob/a248a374fcaac0bf7d2486342be543d4d323fe42/B/blis/build_tarballs.jl#L138